trezorlib/tests/device_tests/test_msg_stellar_sign_transaction.py

```diff
@@ -56,10 +85,71 @@ def test_sign_tx_account_merge_op(self):
 
         tx = self._create_msg()
 
-        response = self.client.stellar_sign_transaction(tx, [op], self.ADDRESS_N, self.get_network_passphrase())
+        response = self.client.stellar_sign_transaction(tx, [op], self.get_address_n(), self.get_network_passphrase())
 
         assert hexlify(response.public_key) == b'15d648bfe4d36f196cfb5735ffd8ca54cd4b8233f743f22449de7cf301cdb469'
-        assert b64encode(response.signature) == b'gjoPRj4sW5o7NAXzYOqPK0uxfPbeKb4Qw48LJiCH/XUZ6YVCiZogePC0Z5ISUlozMh6YO6HoYtuLPbm7jq+eCA=='
+        assert b64encode(response.signature) == b'2R3Pj89U+dWrqy7otUrLLjtANjAg0lmBQL8E+89Po0Y94oqZkauP8j3WE7+/z7vF6XvAMLoOdqRYkUzr2oh7Dg=='
+
+    def test_sign_tx_create_account_op(self):
+        """Create new account with initial balance of 100.0333"""
+        self.setup_mnemonic_nopin_nopassphrase()
+
+        op = proto.StellarCreateAccountOp()
+        op.new_account = stellar.address_to_public_key("GBOVKZBEM2YYLOCDCUXJ4IMRKHN4LCJAE7WEAEA2KF562XFAGDBOB64V")
+        op.starting_balance = 1000333000
+
+        tx = self._create_msg()
+
+        response = self.client.stellar_sign_transaction(tx, [op], self.get_address_n(), self.get_network_passphrase())
+
+        assert b64encode(response.signature) == b'vrRYqkM4b54NrDR05UrW7ZHU7CNcidV0fn+bk9dqOW1bCbmX3YfeRbk2Tf1aea8nr9SD0sfBhtrDpdyxUenjBw=='
+
+    def test_sign_tx_payment_op_native(self):
+        """Native payment of 50.0111 XLM to GBOVKZBEM2YYLOCDCUXJ4IMRKHN4LCJAE7WEAEA2KF562XFAGDBOB64V"""
+        self.setup_mnemonic_nopin_nopassphrase()
+
+        op = proto.StellarPaymentOp()
+        op.amount = 500111000
+        op.destination_account = stellar.address_to_public_key("GBOVKZBEM2YYLOCDCUXJ4IMRKHN4LCJAE7WEAEA2KF562XFAGDBOB64V")
+
+        tx = self._create_msg()
+
+        response = self.client.stellar_sign_transaction(tx, [op], self.get_address_n(),self.get_network_passphrase())
```

Comment:
Missing space between `self.get_address_n(),` and `self.get_network_passphrase()` (that's why the tests are failing).
`./trezorlib/tests/device_tests/test_msg_stellar_sign_transaction.py:117:87: E231 missing whitespace after ','`